### PR TITLE
Add service start script compliance check

### PR DESCRIPTION
**Pull request by rpelisse**
@sabre1041 it's a start and it's not perfect (it only works for SystemD (and it will need to be enhanced to support InitD and be more thorough, but it's probably enough for demo purpose.

### CODE REVIEW

**Comment by rpelisse** [comment]
Weird, this works fine on my machine. @sabre1041 maybe you have an idea how to implement differently so that it's more cross platform and/or resilient ?

File: roles/eap_common_criteria/tasks/service_script_compliance.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  when: 
+    - path_to_jboss_server_service_script.stat.exists
+    - not path_to_jboss_server_service_script.stat.isdir 
+    - not path_to_jboss_server_service_script.stat.islnk
+
+- name: Ensure the jbossas service script used is in compliance
+  assert:
+    that:
+      - output.stdout_lines is defined
+      - output.stdout_lines >= 1
```
Comment (sabre1041) [inline roles/eap_common_criteria/tasks/service_script_compliance.yml:41]: This should be 
`output.stdout_lines |length >= 1`

Comment (rpelisse) [reply roles/eap_common_criteria/tasks/service_script_compliance.yml:41]: ahhhhhhhhhh

**Comment by rpelisse** [comment]
@sabre1041 thanks, fixed :) OK for rebase & merge?

**Review by sabre1041 — APPROVED** [review]

**Pull request merged**